ingestion_tools/infra/sync-v0.0.1.wdl

```diff
@@ -0,0 +1,54 @@
+version 1.0
+
+task cryoet_data_sync_workflow {
+    input {
+        String docker_image_id
+        String aws_region
+        String input_path
+        String input_bucket
+        String output_bucket
+        String output_path
+        String flags
+    }
+
+    command <<<
+        set -euxo pipefail
+        export PYTHONUNBUFFERED=1
+        python --version 1>&2
+        ls -l 1>&2
+        pwd 1>&2
+        cd /usr/src/app/ingestion_tools/scripts
```

Comment:
Are all of these still needed?

Reply:
This file isn't needed yet -- it's for a Coming Soon ™️ feature